src/components/LoanCards/NewHomePageLoanCard.vue

```diff
@@ -73,23 +73,24 @@
 			<kv-loading-paragraph
 				v-if="isLoading"
 				class="tw-mb-1.5 tw-flex-grow"
+				style="min-height: 1rem;"
 			/>
 			<div v-if="!isLoading" class="tw-text-left">
-				<p class="tw-m-0 tw-m-h[3rem] tw-overflow-hidden tw-text-ellipsis tw-line-clamp-2 tw-text-md">
+				<p class="tw-m-0 tw-h-6 tw-m-h[3rem] tw-overflow-hidden tw-text-ellipsis tw-line-clamp-2 tw-text-md">
```

Comment:
Replace JIT class with brackets with inline styles.

Reply:
@dyersituations Could you add examples about this?

Reply:
Actually what is `tw-m-h[3rem]`? I'm not seeing it in the TW docs, so I think it might be incorrect.

I just meant that we can to change TW JIT class, such as `tw-m-[5px]` to the inline style version, or `style="margin: 5px"`. The reason being is that we want to general reduce load time of assets, and each JIT class means an extra class in the compiled TW code that our app loads on EVERY page. So we are working with the idea that removing JIT classes in our somewhat large app could lead to faster load times.

Reply:
JIT stands for "just in time", but just means they are new classes compiled into the TW CSS the app loads.

Reply:
FYI 3rem is a 6 in our tailwind sizing system (1 = 8px; 16px = 1rem). `tw-m-h` isn't a possible class, but if that was meant to be `tw-m-[3rem]` then it should be replaced with `tw-m-6`.